tdi_python/tdicli.py

```diff
@@ -2195,6 +2206,14 @@ def start_tdi(self, in_fd, out_fd, dev_id_list, udf=None, interactive=False, cIn
 
         if hasattr(sys.modules['__main__'], "ipython_app"):
             ipython_app = getattr(sys.modules['__main__'], "ipython_app")
+
+        if hasattr(sys.modules['__main__'], "tdi_session"):
+            tdi_session = getattr(sys.modules['__main__'], "tdi_session")
+        else:
+            sts = create_tdi_python_session()
```

Comment:
Currently, a session is created per device.  Are we changing that? Is there going to be only one session?

Reply:
changed for multi-device